python-usfm-parser/requirements.txt

```diff
@@ -2,3 +2,4 @@ tree-sitter==0.20.0
 jupyterlab==3.4.4
 rnc2rng==2.6.6
 lxml==4.9.1
+pylint==2.15.3
```

Comment:
We should probably have a `dev-requirements.txt` where we put packages that are required only during the development phase so that we don't unnecessarily ship dependencies with the production package.

Reply:
done!